api/s2n.h

```diff
@@ -421,6 +421,97 @@ extern int s2n_connection_set_client_auth_type(struct s2n_connection *conn, s2n_
 S2N_API
 extern int s2n_connection_get_client_cert_chain(struct s2n_connection *conn, uint8_t **der_cert_chain_out, uint32_t *cert_chain_len);
 
+/**
+ * Sets the initial number of session tickets to send after a handshake.
+ *
+ * @param config A pointer to the config object.
+ * @param num The number of session tickets that will be sent.
+ */
+S2N_API
+extern int s2n_config_set_initial_ticket_count(struct s2n_config *config, uint8_t num);
+
+/**
+ * Increases the amount of session tickets to send after a handshake.
+ *
+ * @param conn A pointer to the s2n_connection object.
+ * @param num The number of additional session tickets to send.
```

Comment:
```suggestion
 * @param no_of_additional_session_tickets The number of additional session tickets to send.
```

Reply:
I think the function name is clear enough on its own.